executor/aggfuncs/aggfuncs.go

```diff
@@ -114,6 +115,14 @@ const (
 	DefTimeSize = int64(16)
```

Comment:
This `DefTimeSize` is used for `appendTime` in `countOriginalWithDistinct`, which defines a single 16-bit buf to store time. But `jsonObjectAgg` use `types.Time`, so `DefTimeSize` should be equal to `int64(unsafe.Sizeof(types.Time{}))`. We can define another constant for `countOriginalWithDistinct` and add some comments for it.

Reply:
I have defined `DefDateSize` for `countOriginalWithDistinct`.

Reply:
ok